src/clients/olm_client.py

```
@@ -0,0 +1,128 @@
+                )
+            else:
+                self.client = OllamaApiClient(api_url=api_endpoint)
+
+    @property
+    def _llm_type(self) -> str:
+        return "ollama_client"
+
+    def _call(
+        self,
+        prompt: str,
+        stop: Optional[List[str]] = None,
```

Ruff (0.12.2)

Unused method argument: `stop`

(ARG002)